lua/hop/defaults.lua

```diff
@@ -10,6 +10,7 @@ M.quit_key = '<Esc>'
 M.perm_method = require('hop.perm').TrieBacktrackFilling
 M.reverse_distribution = false
 M.x_bias = 10
+M.custom_distance_fn = nil
```

Comment:
```suggestion
M.distance_method = nil
```
also change the default from `nil` to `manh_dist`.